src/main/java/usersandpostsCRUD/demo/entity/User.java

```diff
@@ -0,0 +1,118 @@
+package usersandpostsCRUD.demo.entity;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+@Table(name = "\"user\"")
+public class User {
+
+    @ManyToOne
+    @JoinColumn(name = "city_id")
+    private City city;
+    @ManyToOne
+    @JoinColumn(name = "country_id")
+    private Country country;
```

Comment:
If user lives in a city, he by default lives in country.

You live in Bitola, which means you are from Macedonia.